### PR TITLE
fix logo cut off

### PR DESCRIPTION
**Pull request by lieblichaaron**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/src/pages/AuthScreens/StartScreen/styles.js [hunk 1]
```diff
@@ -28,14 +28,11 @@ justify-content: center;
 `;
 
 export const LogoContainer = styled.View`
-height: 80px;
-width: 140px;
+width: 100%;
```
Comment (OmerGery) [inline examples/client/Locomotion/src/pages/AuthScreens/StartScreen/styles.js:31]: okay that it's more narrow than before?

**Review by OmerGery — APPROVED** [review]

File: examples/client/Locomotion/src/pages/AuthScreens/StartScreen/styles.js (hunk 1 above)
Comment (OmerGery) [reply examples/client/Locomotion/src/pages/AuthScreens/StartScreen/styles.js:31]: My bad, read it as 100px

Comment (lieblichaaron) [reply examples/client/Locomotion/src/pages/AuthScreens/StartScreen/styles.js:31]: its not more narrow its full width and hight to match according to ratio

Comment (OmerGery) [reply examples/client/Locomotion/src/pages/AuthScreens/StartScreen/styles.js:31]: 👍

**Pull request merged**